frontend/src/static/js/v_summary.js

```diff
@@ -523,12 +528,21 @@ window.vSummary = {
         filtered.push(authorMap[author]);
       });
 
-      filtered.sort(window.comparator((ele) => ele[0][this.sortingOption]));
+      filtered.sort(window.comparator((author) => {
+        if (this.sortingOption === 'totalCommits' || this.sortingOption === 'variance') {
+          return author.reduce(this.groupCommitsVariance, 0);
```

Comment:
the `groupCommitsVariance` now sounds like a property, make it more verby instead, maybe add a get in front.